marblerun-coordinator/values.yaml

```diff
@@ -30,6 +30,14 @@ sgxDevice:
   start: true
 
 
+# webhook configuration
+webhook:
+  start: false
+  webhookImage: ghcr.io/edgelesssys/webhook
+  webhookImageVersion: v0.1.0
```

Comment:
Image versions should be set globally for all parts of marblerun.
E.g. marblerun:v0.3.0 will come with the marble-inector:v0.3.0 image.
If you like you can also refactor the explicit image for each part into setting the registry through values and having a fixed image name for each part. So there is only one place to change if someone would like to use their own registry. See https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/osm-injector-deployment.yaml#L25